### PR TITLE
feat(provider/ecs): Added ScaleDownCluster pipeline stage

### PR DESCRIPTION
**Pull request by BrunoCarrier**
feat(provider/ecs): Added ScaleDownCluster pipeline stage

@anotherchrisberry @icfantv @jrsquared @christopherthielen

### CODE REVIEW

File: app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.scaleDownClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'scaleDownCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./scaleDownClusterStage.html'),
+      executionDetailsUrl: require('./scaleDownClusterExecutionDetails.html'),
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js]: `scaleDownClusterExecutionDetails.html` is an empty file? If you want to use the default, just omit this line.

File: app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      executionConfigSections: ['scaleDownClusterConfig', 'taskStatus'],
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js]: same with `executionConfigSections`, if you just need the default, omit this property.

File: app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.scaleDownClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'scaleDownCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./scaleDownClusterStage.html'),
+      executionConfigSections: ['scaleDownClusterConfig', 'taskStatus'],
```
Comment (jrsquared) [inline app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js]: can remove `executionConfigSections` and the empty `*executionDetails.html` template.

Comment (BrunoCarrier) [reply app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js]: I removed the `executionConfigSections` and the executionDetails.html

**Pull request merged**